createmobilespec/createmobilespec.js

```diff
@@ -303,7 +309,7 @@ var cli = argv.global ? "cordova" : cli_local_bin;
 
 var projectDirName = argv._[0] || "mobilespec";
 var cli_project_dir = path.join(top_dir, projectDirName);
-var browserifyFlag = argv.browserify ? ' --browserify' : '';
+var browserifyFlag = argv.browserify ? ' --browserify' : ''; // TODO
```

Comment:
I just raised PR #153 to resolve this TODO item.

Reply:
This is PR #153. You probably meant #173?

Reply:
> You probably meant #173?

Yes and it is now merged, so this TODO should go away.

Reply:
Would you have proposed the PR to this PR branch, it would have gone automatically...

Reply:
```suggestion
var browserifyFlag = argv.browserify ? ' --browserify' : '';
```